.env.prod-dev

```diff
@@ -5,7 +5,12 @@ DOCS_DOMAIN=http://docs.dev.map.cevi.tools
 FRONTEND_DOMAIN=https://dev.map.cevi.tools
 PRINT_API_BASE_URL=awt-mapfish-print-server
 VALHALLA_DOMAIN=https://valhalla.dev.map.cevi.tools
+STORE_API_URL=http://awt-store-api:6000
 
 # Configures the logging level for the backend
 # valid values are DEBUG, INFO, WARNING, ERROR or CRITICAL
-LOG_LEVEL=INFO
\ No newline at end of file
+LOG_LEVEL=INFO
+
+# MongoDB settings for route storing
+MONGO_INITDB_ROOT_USERNAME=admin
+MONGO_INITDB_ROOT_PASSWORD=pass
```

Comment:
We should consider leaving the password empty and not starting the container without that env variable to force the user to set its own password and don‘t use our default password